src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,171 @@
+---
+title: Get Started
+layout: reference
+---
+
+# Request
+Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including requests creation and transition into the approval workflow.
```

Comment:
... including Requests creation and transition...
(R with uppercase for Request)